inference-engine/cmake/add_ie_target.cmake

```diff
@@ -126,6 +126,11 @@ function(addIeTarget)
             ie_developer_export_targets(${ARG_NAME} ${ARG_EXPORT_DEPENDENCIES})
         endif()
     endif()
+    if(WIN32)
+        # Provide default compile pdb name equal to target name
+        set_target_properties(${ARG_NAME} PROPERTIES COMPILE_PDB_NAME ${ARG_NAME})
```

Comment:
what is PDB name by default?

Reply:
```If you specify a path name that does not include a file name (the path ends in backslash), the compiler creates a .pdb file named VCx0.pdb in the specified directory.```
in our case vc140.pdb for all targets, because on root level cmake we specifiy `CMAKE_COMPILE_PDB_OUTPUT_DIRECTORY` only